server/src/test/java/org/elasticsearch/action/admin/indices/get/GetIndexResponseTests.java

```diff
@@ -112,8 +114,7 @@ private GetIndexResponse createTestInstance(boolean randomTypeName) {
 
     @Override
     protected GetIndexResponse createXContextTestInstance(XContentType xContentType) {
```

Comment:
Can we simplify this test now to have a single `createTestInstance` method, and avoid this extra override?